unit_test/dda/dda.t.c

```diff
@@ -17,7 +17,7 @@
 void	test_dda1(void)
 {
 	t_fpoint	start_pos = {0};
-	t_ray	r = {0};
```

Comment:
why do u remove this?

Reply:
so maybe u can use `ft_bzero()` ?
cuz now `r` will have random values that can cuz bugs that will be very difficult to debug.
btw, we will not submit the tests since they use forbidden func and not in norm, so maybe you can add a compiler comment to ignore this error

Reply:
Ok I try with bzero if not I add a comment